pyk/src/tests/unit/test-data/pyk_toml_test.toml

```diff
@@ -1,6 +1,6 @@
 [coverage]
 output = "default-file"
-definition = "/tmp"
+definition = "/var/folders/ks/1p8zyp5j7xz_v1krhl4l17tm0000gn/T"
```

Comment:
What is this change for?

Reply:
I have not changed that file manually...